pom.xml

```diff
@@ -283,11 +283,25 @@
       <groupId>io.cdap.cdap</groupId>
       <artifactId>hydrator-test</artifactId>
       <version>${cdap.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>io.cdap.cdap</groupId>
+          <artifactId>cdap-api-spark</artifactId>
```

Comment:
how come this was needed?

Reply:
hydrator-test has dependencies on spark_2.10 while in unit tests we use spark_2.11. 
So unless this exlusion is done merged classpath when running unit tests is real mess. And produce a lot of classNotFound etc. pretty common thing for integration tests which are not running on remote instance. We just have to add those excludes to make them work. Due to conflicting versions of dependencies.

Reply:
I see. Ideally you wouldn't have to do this, I've created https://issues.cask.co/browse/CDAP-15323 to simplify this.